src/sql/src/plan/statement/validate.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Queries that validate CONNECTION objects.
+
+use crate::ast::ValidateConnectionStatement;
+use crate::names::Aug;
+use crate::plan::statement::{StatementContext, StatementDesc};
+use crate::plan::{Plan, PlanError, ValidateConnectionPlan};
+
+pub fn describe_validate_connection(
+    _: &StatementContext,
+    _: ValidateConnectionStatement<Aug>,
+) -> Result<StatementDesc, PlanError> {
+    Ok(StatementDesc::new(None))
+}
+
+pub fn plan_validate_connection(
```

Comment:
This should be covered by tests, right? Strangely code coverage is not showing it, I guess a bug in coverage.

Reply:
It's not a bug in coverage, I will add some tests for the `VALIDATE CONNECTION` statements